server/command/server_command_test.go

```
@@ -0,0 +1,327 @@
+	sid, val, ok = parseOptionalServerArg([]string{"/matrix", "server", "map", "srv", "!room:hs"}, 3)
+	require.True(t, ok)
+	assert.Equal(t, "srv", sid)
+	assert.Equal(t, "!room:hs", val)
+
+	// Missing value.
+	_, _, ok = parseOptionalServerArg([]string{"/matrix", "server", "map"}, 3)
+	assert.False(t, ok)
+}
+
+func TestBuildRegistrationYAML(t *testing.T) {
+	server := kvstore.ServerConfig{
```

golangci-lint (2.12.2)

[high] G101: Potential hardcoded credentials

(gosec)